jans-pycloudlib/jans/pycloudlib/persistence/sql.py

```
@@ -1312,3 +1307,49 @@ def override_sql_ssl_property(sql_prop_file):
+    db_name = os.environ.get("CN_SQL_DB_NAME", "jans")
+
+    # In MySQL, physically, a schema is synonymous with a database
+    if db_dialect == "mysql":
+        default_schema = db_name
+    else:  # likely postgres
+        # by default, PostgreSQL creates schema called `public` upon database creation
+        default_schema = "public"
+    return os.environ.get("CN_SQL_DB_SCHEMA", "") or default_schema
+
+
+def set_postgres_search_path(dbapi_connection, connection_record):
```

Ruff (0.14.10)

Unused function argument: `connection_record`

(ARG001)